bigquery/unique_identifiers.md

```diff
@@ -0,0 +1,14 @@
+---
+layout: default
+title: Unique Identifiers
+nav_order: 2
+parent: BigQuery
+---
+
+# Unique Identifiers (DCid)
+
+Every entity in DC has a unique identifier, called ‘dcid’. So, for example, the DCid of California is ['geoId/06'](https://datacommons.org/place/geoId/06) and of India is ['country/IND'](https://datacommons.org/place/country/IND). DCids are not restricted to entities; statistical variables also have DCids, for example the DCid for the Gini Index of Economic Activity is ['GiniIndex_EconomicActivity'](https://datacommons.org/tools/statvar#GiniIndex_EconomicActivity).
```

Comment:
For each new page, can we start with Data Commons (spelled out), before abbreviating it.